strings/src/main/java/com/revenuecat/purchases/strings/AttributionStrings.kt

```diff
@@ -1,5 +1,36 @@
 package com.revenuecat.purchases.strings
 
 object AttributionStrings {
-    const val NO_SUBSCRIBER_ATTRIBUTES_TO_SYNCHRONIZE = "No subscriber attributes to synchronize."
+    const val ATTRIBUTE_ERROR = Emojis.SAD_CAT_EYES + Emojis.ERROR + " There were some subscriber attributes errors: %s"
+    const val ATTRIBUTES_SYNC_SUCCESS = Emojis.HEART_CAT_EYES + " Subscriber attributes synced successfully for " +
+            "App User ID: %s"
+    const val COLLECT_DEVICE_IDS = Emojis.INFO + " collectDeviceIdentifiers called"
+    const val DELETE_ATTRIBUTES = Emojis.INFO + " Deleting subscriber attributes for %s from cache"
+    const val DELETE_ATTRIBUTES_OTHER_USERS = Emojis.INFO + " Deleting old synced subscriber attributes that don't " +
+            "belong to %s"
+    const val INVALID_SUBSCRIBER_ATTRIBUTES = Emojis.SAD_CAT_EYES + Emojis.ERROR + " One or more of the attributes " +
+            "sent could not be saved."
+    const val MARK_ATTRIBUTES_SYNC_SUCCESS = Emojis.INFO + " Marking the following attributes as synced for " +
+            "App User ID: %s"
+    const val NO_SUBSCRIBER_ATTRIBUTES_TO_SYNCHRONIZE = Emojis.INFO + " No subscriber attributes to synchronize."
+    const val SAME_ATTRIBUTES = Emojis.INFO + " Attribution data is the same as latest. Skipping."
+    const val SET_AD = Emojis.INFO + " setAd called"
```

Comment:
these kinda introduce an unfortunate implicit dependency loop:
 - `Purchases.kt` depends on the strings defined in this file
- This file depends on the exact method names defined in `Purchases.kt`. 
If the methods are renamed in `Purchases.kt`, we might never update the strings, since there won't be any compilation issues. 
Maybe we could refactor these so that they take the method name? 
Note that doing so would also mean that we could reuse a single string constant instead of having multiple.

Reply:
Agreed, this could be more general. The string could be "%s called" and in `Purchases.kt` we can indicate the method name.

Reply:
is this being done on a different PR?